Javascript(ES6)/README.md

```diff
@@ -424,7 +424,7 @@ Forked from the excellent [Airbnb JavaScript Style Guide](https://github.com/air
 ## Destructuring
 
   <a name="destructuring--object"></a><a name="5.1"></a>
-  - [5.1](#destructuring--object) Use object destructuring when accessing and using multiple properties of an object. jscs: [`requireObjectDestructuring`](http://jscs.info/rule/requireObjectDestructuring)
+  - [5.1](#destructuring--object) Use object destructuring when accessing and using multiple properties of an object.
```

Comment:
I thought it did not exist (just because the original guide didn't have it), but I looked around and it actually does. I added it to the .eslintrc file and the reference to this line.